src/components/StaticFilters.tsx

```diff
@@ -0,0 +1,133 @@
+import { useMemo } from 'react';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import {
+  ApplyFiltersButton,
+  CheckboxOption,
+  CollapsibleLabel,
+  CollapsibleSection,
+  FilterGroup,
+  SearchInput
+} from './Filters';
+import { StaticFilters as StaticFiltersCompoundComponent } from './Filters/StaticFilters';
+
+/**
+ * The CSS class interface for {@link StaticFilters}.
+ *
+ * @public
+ */
+export interface StaticFiltersCssClasses {
+  container?: string,
+  searchInput?: string,
+  optionsContainer?: string,
+  option?: string,
+  optionInput?: string,
+  optionLabel?: string,
+  applyFiltersButton?: string
+}
+
+/**
+ * The configuration data for a filter option.
+ *
+ * @public
+ */
+export interface FilterOptionConfig {
+  /** {@inheritDoc Filters.CheckboxOptionProps.value} */
+  value: string | number | boolean,
+  /** {@inheritDoc Filters.CheckboxOptionProps.label} */
+  label?: string,
+  /** {@inheritDoc Filters.CheckboxOptionProps.selectedByDefault} */
+  selectedByDefault?: boolean
+}
+
+/**
+ * Props for the {@link StaticFilters} component.
+ *
+ * @public
+ */
+export interface StaticFiltersProps {
+  /** The fieldId corresponding to the static filter group. */
+  fieldId: string,
+  /** {@inheritDoc FilterOptionConfig} */
+  filterOptions: FilterOptionConfig[],
+  /** The displayed label for the static filter group. */
+  title?: string,
+  /** Whether or not the filter is collapsible. Defaults to true. */
+  collapsible?: boolean,
+  /**
+   * If the filter group is collapsible, whether or not it should start out
+   * expanded. Defaults to true.
+   */
+  defaultExpanded?: boolean,
+  /** Whether or not to display a text input to search for filter options. */
+  searchable?: boolean,
+  /**
+   * Whether or not a search is automatically run when a filter is selected.
+   * Defaults to true.
+   */
+  searchOnChange?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: StaticFiltersCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
```

Comment:
Is it intentional that `cssCompositionMethod` only applies to CheckboxOption here? I feel like that's a little confusing.

Reply:
It is intentional because the other compounds components, besides `CheckboxOption`, set their styling with a single `className` prop, so they don't accept a composition method. I think the only case where the behavior might differ from what's expected is for the `merge` composition method because the `className` would still be replaced instead of merged. `assign` and `replace` would match current behavior even without passing along the composition method.

Since that could be confusing, I could update the docs and/or name it something like `optionsCssCompositionMethod` to make it clear that it's only for the checkbox options. Alternatively, since it still might be a little weird to have a composition method for only some of the CSS classes, maybe we could compose all of the classes before passing them to the different components. The downside of that would be some additional complexity and we'd need to export the default styling for the compound components with the single `className` prop (they're currently just set inline as a default for the prop).

Reply:
I like the idea floating around in slack of getting rid of cssCompositionMethod and always having it be merge